<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDevPosCtl1</class>
 <widget class="QWidget" name="CDevPosCtl1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>721</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="CKeyComboBox" name="comboBox_line">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CKeyComboBox" name="comboBox_machine">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="CKeyCheckBox" name="checkBox_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_7">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_12">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>10</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>12</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_11">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_10">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>10</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="CKeyCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKeyComboBox</class>
   <extends>QComboBox</extends>
   <header>ckeycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CKeyCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ckeycheckbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_line</tabstop>
  <tabstop>comboBox_machine</tabstop>
  <tabstop>checkBox_1</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>checkBox_12</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
